InterviewFlashCards/QueryManager.swift

```diff
@@ -1,12 +1,11 @@
-
 import Foundation
 import Firebase
 import FirebaseDatabase
 
-public typealias JSON = [[String: Any]]
-public protocol Networking {
+public typealias JSON = [String: Any]
```

Comment:
ahhh, gotcha! i missed that my original question was asking the purpose of the `[]` in the closure. why do it that way vs keeping it as an array of dictionaries?